packages/block-editor/src/components/block-patterns-list/index.js

```diff
@@ -101,10 +114,16 @@ function BlockPattern( {
 								pattern.description ? descriptionId : undefined
 							}
 						>
-							<BlockPreview
-								blocks={ blocks }
-								viewportWidth={ viewportWidth }
-							/>
+							{ ! isEmpty ? (
+								<BlockPreview
```

Comment:
Should we also disable the `onClick` on the `.block-editor-block-patterns-list__item` element? 

If you click on a pattern with no content, a message appears — _"[Your pattern] inserted"_ —  but it inserts nothing.